src/main/resources/application.yml

```diff
@@ -8,6 +8,8 @@ spring:
   codec:
     max-in-memory-size: 10MB
 
+  mvc.async.request-timeout: 200000
```

Comment:
Increasing the default timeout for async web requests - doesn't seem to be a way to do this on a per request basis.

Reply:
@Mjwillis would increasing this cause an issue - as this will affect other calls?

Reply:
It hopefully shouldn't have an effect elsewhere - nothing else is running async and even if it was the only side effect would be it would allow some calls to run for a bit longer